proto/store/setting.proto

```diff
@@ -33,6 +33,28 @@ message WorkspaceProfileSetting {
 
   // The duration for refresh token.
   google.protobuf.Duration refresh_token_duration = 6;
+
+  // The setting of custom announcement
+  Announcement announcement = 7;
+}
+
+message Announcement {
+  // We support three levels of AlertLevel: INFO, WARNING, and ERROR.
+  enum AlertLevel {
+    ALERTLEVEL_UNSPECIFIED = 0;
+    ALERTLEVEL_INFO = 1;
+    ALERTLEVEL_WARNING = 2;
+    ALERTLEVEL_CRITICAL = 3;
```

Comment:
```suggestion
    ALERT_LEVEL_UNSPECIFIED = 0;
    ALERT_LEVEL_INFO = 1;
    ALERT_LEVEL_WARNING = 2;
    ALERT_LEVEL_CRITICAL = 3;
```

Reply:
Please use the same naming style. You can refer the enum definition in `common.proto`.

Reply:
Following https://protobuf.dev/programming-guides/proto3/#enum, ALERTLEVEL_XXX maybe be better.